src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+  let discount = 0;
+
+  if (days >= 7) {
+    discount = -50;
+  } else if (days >= 3) {
+    discount = -20;
+  }
```

Comment:
50 and 20 are the magic numbers, create variables for them, like you did for cost